### ISSUE TITLE
Update default columns to show intake latest (replace updated date col)

### ISSUE THREAD

**Issue by cazcam34** [talentbeyondboundaries/tbbtalentv2#1327]
- [x] Add IELTS score to default columns, remove updated date from default 
- [x] Add 'Additional Information' section to general tab on candidate search card

Remainder of issue #1253

**Comment by cazcam34** [comment]
Added 'Additional Informaiton section to search card - it was missing as were weren't passing in an 'adminUser' input which that section requires in order to view. In fact there was a few missing inputs on tab sections. I fixed this up and matched the tab permission inputs (adminUser) on all tabs and now the permissions are the same for the search card and the view.

However there are two auth methods used for permissions: 
1. isAdminUser which just checks for admin role
2. canViewPrivateInformation this checks for admin role but also that the partner matches the candidate partner (or if default partner can view all as long as admin). Is this the better auth method to use instead of isAdminUser?
